ecs_fargate/datadog_checks/ecs_fargate/ecs_fargate.py

```diff
@@ -182,20 +182,30 @@ def submit_perf_metrics(self, container_tags, container_id, container_stats):
             cpu_stats = container_stats.get('cpu_stats', {})
             prev_cpu_stats = container_stats.get('precpu_stats', {})
 
-            value_system = cpu_stats.get('system_cpu_usage')
+            value_system = cpu_stats.get('cpu_usage', {}).get('usage_in_kernelmode')
             if value_system is not None:
-                self.gauge('ecs.fargate.cpu.system', value_system, tags)
+                self.rate('ecs.fargate.cpu.system', value_system, tags)
+
+            value_user = cpu_stats.get('cpu_usage', {}).get('usage_in_usermode')
+            if value_user is not None:
+                self.rate('ecs.fargate.cpu.user', value_user, tags)
 
             value_total = cpu_stats.get('cpu_usage', {}).get('total_usage')
             if value_total is not None:
-                self.gauge('ecs.fargate.cpu.user', value_total, tags)
+                self.rate('ecs.fargate.cpu.usage', value_total, tags)
 
+            available_cpu = cpu_stats.get('system_cpu_usage')
```

Comment:
This is a bit confusing to me. I expected "system_cpu_usage" to represent "usage" instead of "available" 🤔

Reply:
Yes I know, that's why I renamed it, the name in ECS API output is misleading.